src/AutofacSerilogIntegration/AutofacSerilogIntegration.csproj

```diff
@@ -1,9 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Description>Contextual logger injection for Autofac using Serilog</Description>
     <AssemblyTitle>AutofacSerilogIntegration</AssemblyTitle>
-    <VersionPrefix>4.0.1</VersionPrefix>
+    <VersionPrefix>5.0.0-alpha2</VersionPrefix>
```

Comment:
The `-alpha2` suffix isn't needed here, prerelease tags are added automatically by the build

Reply:
Fixed.